lib/travis/model/repository.rb

```diff
@@ -39,8 +39,6 @@ class Repository < Travis::Model
   scope :by_params, ->(params) {
     if id = params[:repository_id] || params[:id]
       where(id: id)
-    elsif params[:github_id]
-      where('vcs_id = :id OR github_id = :id_i', id: params[:github_id].to_s, id_i: params[:github_id].to_i)
     elsif params[:vcs_id] && params['vcs_type']
```

Comment:
I think that its more safe to keep condition with params[:vcs_id] and empty vcs_type => Github is default